presto-docs/src/main/sphinx/presto_cpp/properties.rst

```diff
@@ -95,14 +95,43 @@ The configuration properties of Presto C++ workers are described here, in alphab
 
   In-memory cache.
 
-``query.max-memory-per-node``
+``runtime-metrics-collection-enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* **Type:** ``boolean``
+* **Default value:** ``false``
+
+  Enables collection of worker level metrics.
+
+``task.max-drivers-per-task``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 * **Type:** ``integer``
+* **Default value:** ``number of hardware CPUs``
+
+  Number of drivers to use per task. Defaults to hardware CPUs.
+
+``query.max-memory-per-node``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``string``
 * **Default value:** ``4GB``
 
   Max memory usage for each query.
 
+
+``system-memory-gb``
+^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``integer``
+* **Default value:** ``40``
+
+  Memory allocation limit enforced via internal memory allocator. It consists of two parts:
```

Comment:
```suggestion
  Memory allocation limit enforced by a internal memory allocator. It consists of two parts:
```
See the [Gitlab documentation style guide recommended word list entry for via](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#via) for discussion and alternate suggestions if my suggestion here doesn't work for you.